ci/emg_api_datafiles/results/crates/motus/1_motus_SRR8845517/page2.html

```diff
@@ -0,0 +1,5 @@
+<h1>
```

Comment:
We shouldn't need the unzipped crate contents in the test fixtures – just the .zip. Similarly, we probably don't need 3 versions, 1_motus, 2_.... unless there is something different about them?

Reply:
I see that 3 of these zips are in the `RUN_DOWNLOAD` in the emg-testdb.sqlite fixture too, so just check whether all 3 are really needed for minimal dev/testing.

Reply:
@SandyRogers I've created a pull request for the emgapi changes here:
https://github.com/EBI-Metagenomics/emgapi/pull/322

Reply:
@SandyRogers I initially created the extra mOTUs folders, just to see that the singleton was working as expected when there were multiple entries. They are no longer needed and have now been removed.